components/02-elements/rating/mocks/ratingItems.json

```diff
@@ -0,0 +1,34 @@
+{
+  "items": [
```

Comment:
I'm not sure what for is this config... All we need to define is a number of stars and then get or set the rating.